scripts/startup.sh

```diff
@@ -31,22 +31,15 @@ UNIVERSE_DOMAIN="$($CURL $URL/instance/attributes/universe_domain)"
 STORAGE_CMD="CLOUDSDK_CORE_UNIVERSE_DOMAIN=$UNIVERSE_DOMAIN gcloud storage"
 
 function devel::zip() {
-	local BUCKET="$($CURL $URL/instance/attributes/slurm_bucket_path)"
-	if [[ -z $BUCKET ]]; then
-		echo "ERROR: No bucket path detected."
-		return 1
-	fi
-
 	local SLURM_ZIP_URL="$BUCKET/slurm-gcp-devel.zip"
 	local SLURM_ZIP_FILE="$HOME/slurm-gcp-devel.zip"
-	local SLURM_ZIP_DIR="$HOME/slurm-gcp-devel"
 	eval $(bash -c "$STORAGE_CMD cp $SLURM_ZIP_URL $SLURM_ZIP_FILE")
 	if ! [[ -f "$SLURM_ZIP_FILE" ]]; then
-		echo "INFO: No development files downloaded. Skipping."
```

Comment:
Do we want to do this?  Does this file always run?  What if users mark enable_devel: False?  Does it skip running the whole script?  If this isn't always run then my config stuff will probably break as well.

Reply:
> What if users mark enable_devel: False

They can't do it, "devel" is always on. There is only one correct set of scripts - one that is in devel tarball.

Reply:
> Does it skip running the whole script?

It fails startup.

> If this isn't always run then my config stuff will probably break as well.

It's OK, we want to fail in this case.